.githooks/commit-msg/add-random-emoji

```diff
@@ -13,6 +13,6 @@ fi
 # POSITION=$(($(od -A n -t d -N 1 /dev/urandom | tr -d ' ') % 11 + 1))
 SELECTED=":anchor:" #$(echo "$EMOJIS" | awk -v N=$POSITION '{ print $N }')
 
-sed -i -E '1s/(.*)\s*:\w+:)/\1/' "$1"
+sed -i -E '1s/(.*)\s*:\w+:/\1/' "$1"
```

Comment:
Saw this error popping up in my terminal when executing this hook

Reply:
Jeah its already fixe on another branch!, Thx :)!